console/src/tasks.rs

```diff
@@ -105,6 +110,10 @@ impl State {
     }
 
     pub(crate) fn update_tasks(&mut self, update: proto::tasks::TaskUpdate) {
+        if let Some(now) = update.now {
+            self.last_updated_at = Some(now.into());
+        }
+        
         let mut stats_update = update.stats_update;
```

Comment:
oh okay, i guess rustfmt doesn't agree with me about this?
```suggestion
        }
        let mut stats_update = update.stats_update;
```